components/fieldgroups/experience-event/experienceevent-media-collection-downloaded.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/experienceevent-media-collection-downloaded",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "MediaCollection Interaction Details ",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:tags": {
+    "industry": ["all"]
+  },
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Track interactions with media collection downloaded.",
+  "definitions": {
+    "experienceevent-mediaCollectionDownloaded": {
+      "properties": {
+        "xdm:events": {
```

Comment:
"event" as such has a very genetic meaning on the platform. It can easily be confused by users for any event. I would advise to prefix this with some context e.g : "xdm:mediaEvents" ? Also, I have already left a comment in the mediaEvent data type. It is referred here in the array. You will have to fix that as that is currently defined as a class and not an array.

Reply:
Good point. I added a new commit with this change of renaming `events` to `mediaEvents`. Thank you!